config.json

```diff
@@ -1,661 +1,674 @@
 {
+  "language": "C",
```

Comment:
Here you've made a whole bunch of changes to the config.json. Instead it would be best to update _just_ the part that relates to the exercise you are implementing, leaving the other changes to a PR specifically for updating the config.